cilium/cmd/bpf_egress_delete.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/maps/egressmap"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	egressDeleteUsage = "Delete egress entries using source IP and destination CIDR.\n"
+)
+
+var bpfEgressDeleteCmd = &cobra.Command{
+	Args:  cobra.ExactArgs(2),
+	Use:   "delete",
```

Comment:
It's sometimes useful to have a `flush` or `delete --all` command to flush the local state during incidents or when debugging. Can be done as a follow up though.

Reply:
We could track this at https://github.com/cilium/cilium/issues/13575 as well, though probably not required for 1.10.